### ISSUE TITLE
Timeline button failing Lighthouse contrast test for accessibility

### ISSUE THREAD

**Issue by palewire** [mastodon/mastodon#21239]
### Steps to reproduce the problem

Go to your home timeline in the light theme. Run a Lighthouse Accessibility test. You will see that the Publish button and notification bubble fail a contrast test.

### Expected behaviour

Pass contrast test

### Actual behaviour

Fails contrast test

### Detailed description

Here's the test results.

![Screenshot from 2022-11-20 09-11-52](https://user-images.githubusercontent.com/9993/202916018-03fc4a0f-30c5-419e-a45a-839f3a03989f.png)

This is confirmed by running the colors through a contrast test.

![Screenshot from 2022-11-20 09-09-47](https://user-images.githubusercontent.com/9993/202915961-e410450f-702b-48c5-b2b2-ccb3b5dfe673.png)

Thankfully, the Mastodon style guide includes a lighter version of purple.

![Screenshot from 2022-11-20 09-11-31](https://user-images.githubusercontent.com/9993/202916034-96f8b008-9d85-4241-8958-707155a0ebe2.png)

Try that in a contrast test with black text and it passes both the AA and AAA standard.

![Screenshot from 2022-11-20 09-11-20](https://user-images.githubusercontent.com/9993/202916068-c2142486-7a7f-4bfa-87ec-c972e978a713.png)

If the maintainers agree, I propose changing the color of the button and copy to the passing colors. This may require other complimentary changes to other elements. I'd be happy to contribute the code, if you'd like the pull request.

### Specifications

Brave browser

**Comment by daguar** [comment]
I'll just add that I saw a report from a user today of a similar/related issue with the color scheme of the "repost" button, stating that because their sight is "color deficient" it is very difficult for them to see in the default web view whether they have reposted something or not.

Here is the visual difference right now:

<img width="559" alt="Screen Shot 2022-12-30 at 9 01 29 AM" src="https://user-images.githubusercontent.com/994938/210095395-c947dc6c-e9a9-431b-b5d1-f94b09174f7f.png">
<img width="556" alt="Screen Shot 2022-12-30 at 9 01 35 AM" src="https://user-images.githubusercontent.com/994938/210095398-116d07fc-e352-47ad-a134-483f6a7a4ad6.png">
